topics/transcriptomics/tutorials/ref-based/tutorial.md

```diff
@@ -1387,7 +1390,7 @@ DESeq2 requires to provide for each factor, counts of samples in each category.
 >            - *"Find pattern"*: `(.*)_(.*)_(.*)`
 >            - *"Replace with"*: `\1_\2_\3\tgroup:\2\tgroup:\3`
 >
->     This step adds an additional columns that can be used with the ``Tag elements from file`` tool
+>     This step adds an additional columns that can be used with the **Tag elements from file** {% icon tool %} tool
```

Comment:
```suggestion
>     This step adds an additional columns that can be used with the {% tool [Tag elements from file](__TAG_FROM_FILE__) %}  tool
```

`{% tool Tag Elements from file %}` should work as equivalent syntax and makes sure you get it right, but we can make it better by linking the tool maybe. https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#tool-links

Reply:
Oups, I forgot there was a commit suggestion button...